src/main/resources/standardTestCases/globbingBehavior/globbingBehavior.wdl

```diff
@@ -0,0 +1,74 @@
+task writeFiles {
+  String dir
+  command <<<
+    mkdir ${dir}
+    echo "hullo" > ${dir}/hello.txt
+
+    echo "what" > what.txt
+    echo "how" > how.txt
+    echo "why" > why.txt
+
+    cd ${dir}
+    echo "buh-bye" > ciao.txt
+
+    sleep 2
+  >>>
+  output {
+    Array[File] strInterpolation=  glob("${dir}/*.txt")
+    Array[File] callDirectory = glob("*.txt")
+    Array[File] specificDirectory = glob(dir + "/*.txt")
+  }
+  runtime {
+    docker:"ubuntu:latest"
+  }
+}
+
+task unexpectedReturnCode {
+
+  command {
+    echo "easiest glob job ever!" > tiny.txt
+
+    sleep 2
+    exit 3
+  }
+
+  output {
+    Array[File] easyGlob = glob("*.txt")
+  }
+
+  runtime {
+    docker: "ubuntu:latest"
+    continueOnReturnCode: 3
+  }
+}
+
+workflow globbingBehavior {
+  String outputDir = "out"
+
+  call writeFiles { input: dir = outputDir }
+
+  call unexpectedReturnCode {}
+
+  scatter (f in writeFiles.strInterpolation) {
+    String strInterpolation_contents = read_string(f)
```

Comment:
Do we really want to officially endorse "you can use 'scatter' as a 'map' function" in our test cases?